### ISSUE TITLE
Playground currently broken on Windows

### ISSUE THREAD

**Issue by snth** [PRQL/prql#3857]
### What happened?

![image](https://github.com/PRQL/prql/assets/652285/2d69c1f3-bce0-4567-b8e9-531cfcc0bad9)

### PRQL input

```elm
# Default example
```

### SQL output

```SQL
# Nothing
```

### Expected SQL output

```SQL
# Default example
```

### MVCE confirmation

- [X] Minimal example
- [X] New issue

**Comment by richb-hanover** [comment]
Hmmm... Both the public Playground and the latest Dev Container e798ddc7 work fine for me on macOS 12.6.7. I saw both a PRQL query on the left and a bunch of SQL on the right. (I did not verify the SQL code.)

I tried:

* Chrome Version 120.0.6099.35 (Official Build) beta (x86_64)
* Edge Version 119.0.2151.72 (Official build) (x86_64)
* Firefox 120.0 (64-bit)
* Safari Version 17.1 (17616.2.9.11.9, 17616)

**Comment by snth** [comment]
Thanks for trying to reproduce this @richb-hanover .

I'm on Windows 10 Pro (Version 10.0.19045 Build 19045). 

My browser versions are:
- Chrome Version 119.0.6045.160 (Official Build) (64-bit)
- Edge 
  - Initially: Version 119.0.2151.58 (Official build) (64-bit)
  - After upgrade: Version 119.0.2151.93 (Official build) (64-bit)

I can't get any SQL output on any of those.

**Comment by richb-hanover** [comment]
And which version/commit of the Playground are you using? Thanks.

**Comment by max-sixty** [comment]
That's quite weird. It works fine for me on both Firefox & Chrome, on MacOS.

@snth do you see anything in the dev tools console?

**Comment by snth** [comment]
Yeah, so I've been looking into it with the dev tools console. There are two requests that fail/are blocked by my ad blocker (pi-hole):

1. https://www.googletagmanager.com/gtag/js?id=G-LQJDD599T4
2. https://cdn.jsdelivr.net/npm/@duckdb/duckdb-wasm@1.28.0/dist/duckdb-browser-eh.worker.js

I added whitelist rules for:
1. googletagmanager.com
2. cdn.jsdelivr.net

Interestingly, the first whitelist rule was ineffective and that particular request still fails. I thought it couldn't be the second one because while the "Query Results" panel may fail, that shouldn't block the "Compiled SQL" panel. However once I added the cdn whitelist rule the playground now works.

---
I then removed the whitelist rules again just to prove my thesis but everything keeps working. I figured maybe the files are in my browser cache so I cleared the browser cache but it's still working. I guess maybe my ISP is caching some of that CDN content. I'll wait for it to fail again and then try again.

---
Bottom line: given that many people use ad blockers, I'm relieved that having the googletagmanager/Google analytics blocked doesn't affect things. To me it's a bit odd that having the DuckDB wasm file blocked breaks the whole playground.

My pi-hole version is probably out of date since it probably shouldn't be blocking a JS and NPM CDN. I guess we can close this issue.

**Comment by max-sixty** [comment]
OK cool! 

---

I would prefer if `duckdb-wasm` bundled everything it needs rather than reaching out to a CDN, but I guess that's out of our control...
